lib/tldr.js

```diff
@@ -193,13 +157,14 @@ function renderContent(content, options={}) {
 }
 
 function checkStale() {
-  cache.lastUpdated((err, stats) => {
-    if (err) {
+  cache.lastUpdated()
+    .then(stats => {
```

Comment:
Please use `()` around `stats`. Applies everywhere.

Reply:
Ok, tried to go along with established conventions but missed this one 😞 
IMO it should be checked using eslint: https://eslint.org/docs/rules/arrow-parens

Reply:
Fully agree. Possibly something as a next PR ? :smile:

Reply:
Sure 😉